<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="608" started-at="2018-10-16T07:27:39Z" finished-at="2018-10-16T07:27:39Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="608" started-at="2018-10-16T07:27:39Z" finished-at="2018-10-16T07:27:39Z">
      <class name="com.test.ExtentReport">
        <test-method status="PASS" signature="startReport()[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="startReport" is-config="true" duration-ms="202" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="FAIL" signature="login()[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="login" duration-ms="7" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.test.ExtentReport.login(ExtentReport.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="generateResults(org.testng.ITestResult)[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="generateResults" is-config="true" duration-ms="4" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=ExtentReport.login()[pri:0, instance:com.test.ExtentReport@2f7c7260] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateResults -->
        <test-method status="SKIP" signature="mobileRecharge()[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="mobileRecharge" duration-ms="0" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[This is not ready for testing]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: This is not ready for testing
	at com.test.ExtentReport.mobileRecharge(ExtentReport.java:93)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mobileRecharge -->
        <test-method status="PASS" signature="register()[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="register" duration-ms="1" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- register -->
        <test-method status="PASS" signature="generateResults(org.testng.ITestResult)[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="generateResults" is-config="true" duration-ms="0" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=mobileRecharge status=SKIP method=ExtentReport.mobileRecharge()[pri:0, instance:com.test.ExtentReport@2f7c7260] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateResults -->
        <test-method status="PASS" signature="generateResults(org.testng.ITestResult)[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="generateResults" is-config="true" duration-ms="0" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=register status=SUCCESS method=ExtentReport.register()[pri:0, instance:com.test.ExtentReport@2f7c7260] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateResults -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.test.ExtentReport@2f7c7260]" name="tearDown" is-config="true" duration-ms="375" started-at="2018-10-16T12:57:39Z" finished-at="2018-10-16T12:57:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.ExtentReport -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
